Xwt.WPF/Xwt.WPFBackend/AccessibleBackend.cs

```diff
@@ -1,18 +1,17 @@
 using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+using System.Threading.Tasks;
```

Comment:
Usings are unrelated here, we can clean them up, but adding new ones doesn't make sense.